src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public interface FruitService {
+    void getOperationByFruit(List<FruitTransaction> fruitTransactionList,
```

Comment:
1. rename the method to mirror its logic. 'get...' method that returns nothing looks very confusing
2. strategy is an implementation detail, it shouldn't be used in the interface. consider making it class variable